### DISCUSSION TITLE
Planet Imagery zoom and split feature

### DISCUSSION THREAD

**Discussion by Niko-La** [opengeos/leafmap#147] [Q&A] [answered]
```
os.environ["PLANET_API_KEY"] = "######"
monthly_tiles = leafmap.planet_monthly_tiles()

leafmap.ts_inspector(monthly_tiles)
```

1) is it possible to select left and right tiles for specific year and date?
2) is it possible to use 'center and 'zoom' the planet imaginary 

if not a potential future feature?

**Comment by giswqs** [comment, marked as answer]
Yes, you can. 
```python
leafmap.ts_inspector(monthly_tiles, left_name="Planet_2020_09", right_name="Planet_2020_10", width="130px", center=[30, -90], zoom=10)
```
![image](https://user-images.githubusercontent.com/5016453/147023102-099e6a37-afd0-4e99-b618-0cb615938d80.png)

**Reply by giswqs** [reply]
I just added documentation for the function. See https://leafmap.org/leafmap/#leafmap.leafmap.ts_inspector

**Comment by Niko-La** [comment]
worked great.

in that case a bollean flag to drop a pin at the location would help with tracking>


To get more accurate maps we need to enable it through Planetlab dashboard and should be able to see int `leafmap.planet_monthly()`
and reference the layer name in left and right name options?
I think the free trial samples maybe at best 50cm resolution?

**Comment by giswqs** [comment]
You can use `m.add_marker_cluster()` to add markers to the map. It won't work on a split-panel map. 

![Peek 2021-12-21 22-39](https://user-images.githubusercontent.com/5016453/147031607-c188ac6f-07aa-4aeb-bd3e-5c2eb2b33c87.gif)

**Reply by Niko-La** [reply]
ah so no pin markets on split. How about the `add_circle_markers_from_xy`   or any other way to show a marker or object on split?

**Reply by giswqs** [reply]
Unfortunately, the split-panel map won't support adding additional data layers (e.g., markers, geojson, heatmap). This is a limitation of ipyleaflet. Leafmap can't support this until it has been implemented by ipyleaflet. Feel free to open an issue on the ipyleaflet repo. 

https://github.com/jupyter-widgets/ipyleaflet/issues/440
https://github.com/jupyter-widgets/ipyleaflet/issues/532
